pandas/_libs/tslibs/np_datetime.pxd

```diff
@@ -5,12 +5,67 @@ from cpython.datetime cimport date, datetime
 
 from numpy cimport int64_t, int32_t
 
+cdef extern from "numpy/ndarrayobject.h":
+    ctypedef int64_t npy_timedelta
+    ctypedef int64_t npy_datetime
+
+cdef extern from "numpy/ndarraytypes.h":
+    # ctypedef struct npy_datetimestruct:
+    #    int64_t year
+    #    int32_t month, day, hour, min, sec, us, ps, as
+
+    ctypedef enum NPY_DATETIMEUNIT:
```

Comment:
Is this enum needed?

Reply:
No, it's leftover from an earlier revision.  Will remove.